x-pack/solutions/security/plugins/security_solution/common/detection_engine/prebuilt_rules/diff/extract_threat_array.test.ts

```diff
@@ -12,6 +12,59 @@ import { extractThreatArray } from './extract_threat_array';
 const mockThreat = getThreatMock()[0];
 
 describe('extractThreatArray', () => {
+  it('normalizes url ending backslashes', () => {
+    const mockRule = {
+      ...getRulesSchemaMock(),
+      threat: [
+        {
+          ...mockThreat,
+          tactic: {
+            ...mockThreat.tactic,
+            reference: 'https://attack.mitre.org/tactics/TA0000',
+          },
+          technique: [
+            {
+              ...mockThreat.technique![0],
+              reference: 'https://attack.mitre.org/techniques/T0000',
```

Comment:
It'd be nice to have one `reference` with a backslash here or in a separate test.